tools/downloader/common.py

```diff
@@ -51,6 +51,7 @@
 }
 KNOWN_TASK_TYPES = {
     'action_recognition',
+    'sound_classification',
```

Comment:
Please move this so that the list remains sorted (ditto for the `README`).

Reply:
Done.